1-exercises/D-for-in-loops/exercise-2.js

```diff
@@ -24,7 +24,13 @@ let shapes = {
 let shapesWithMoreThanFiveSides = [];
 
 // WRITE YOUR CODE BELOW THIS LINE
-
+for (const key in shapes) {
```

Comment:
Did you run the tests for this? This is not quite right.

If you read the question it asks you to 'construct an array' and that shapesWithMoreThanFiveSides has already been declared for you.

You need to use this array. If you look ast the expected output you will also see that a) you are not retuning the keys and b) you are missing one of the shapes